README.md

```diff
@@ -153,6 +154,14 @@ For more information see: http://www.scala-sbt.org/0.13/docs/Testing.html#Test+F
 
 
 ### IntelliJ
+We now recommend opening a separate IntelliJ project for the automation test code. From 
+IntelliJ choose File->New->Project From Existing Sources and select automation/build.sbt.
```

Comment:
You might want to mention that you'll have to accept the offer from intellij to add the git root.